tasks/UsblDock.cpp

```diff
@@ -29,14 +29,11 @@ bool UsblDock::configureHook()
     if (! UsblDockBase::configureHook())
         return false;
 
-    if (!_io_port.get().empty())
+    if(_interface.get() != ETHERNET)
```

Comment:
Good point.
I put the property because Evologics can provide a device with 2 connectors interface (rs232 and ethernet), so the driver would be more "generic". 
By the other hand our usbl has just one connector, so I added the check just be sure there is no confusion about it.
So, would be better to make the driver less "generic" and remove property, or to remove specific check? 
I think removing specific check would be a better option, and the user had to be sure about the interface.